src/app/App.tsx

```diff
@@ -1,85 +1,38 @@
-import React, { useState } from 'react'
-import { BrowserRouter, Routes, Route } from 'react-router-dom'
+
+import React, { useEffect, useState } from 'react'
+import { BrowserRouter, Routes, Route, useParams } from 'react-router-dom'
+import TopicView from './pages/TopicView/TopicView'
 import Dashboard from './pages/Dashboard/Dashboard'
 
 import type { Need, Topic } from './types/types'
+import type { ObjectId } from 'mongodb'
```

Comment:
Don't use this type in the frontend. I would suggest just declaring the id as a string